packages/terra-table/docs/TableHeaderCell.md

```diff
@@ -0,0 +1,26 @@
+# Terra Table Header Cell
+
+The Terra Table Header Cell is a sub component of the table header. The header cell dictates the minimum width of the column, but it can also dictate the maximum width of the column if a fixed or max-width value is set it. This results in the minimum width of the content also being the maximum width of the column.
+
```

Comment:
> This results in the minimum width of the content also being the maximum width of the column.

If I set a minWidth here, that means its also the maxWidth? Did I read that correctly?

Reply:
Removed confusing text.